spras/runner.py

```diff
@@ -42,6 +43,15 @@ def merge_input(dataset_dict, dataset_file):
     dataset = Dataset(dataset_dict)
     dataset.to_file(dataset_file)
 
+def merge_gold_standard_input(gs_dict, gs_file):
```

Comment:
This doesn't depend on `algorithm` as an input, so I don't think it needs to go in `runner.py`. Should it be in `evaluation.py`?